src/main/java/uk/gov/hmcts/reform/prl/controllers/EditAndApproveDraftOrderController.java

```diff
@@ -215,7 +214,7 @@ public AboutToStartOrSubmitCallbackResponse populateCommonFields(
                 CaseData.class
             );
             Map<String, Object> response = draftAnOrderService.populateCommonDraftOrderFields(authorisation, caseData);
-            String errorMessage = draftAnOrderService.checkIfOrderCanReviewed(callbackRequest, response);
+            String errorMessage = DraftAnOrderService.checkIfOrderCanReviewed(callbackRequest, response);
```

Comment:
What is the change here? DraftAnOrderService instead of draftAnOrderService? but other service calls are still having draftAnOrderService

Reply:
the method which we are calling here is static method.So we are calling with the class instead of object